deploy/platform/docker/docker-compose.mysql.yaml

```diff
@@ -53,20 +53,20 @@ services:
       - sw
     depends_on:
       - mysql
-    
+
 
   otel-collector:
     image: otel/opentelemetry-collector:0.50.0
     networks:
       - sw
-    command: [ "--config=/etc/mysql-otel-collector-config.yaml" ]
+    command: [ "--config=/etc/otel-collector-config.yaml" ]
     volumes:
-      - ./config/mysql-otel-collector-config.yaml:/etc/mysql-otel-collector-config.yaml
+      - ./config/otel-collector-config-mysql:/etc/otel-collector-config.yaml
```

Comment:
Hello , I'm working on apisix showcase  https://github.com/apache/skywalking/issues/9677 .
Did the modification miss file  extension `yaml` ?

Reply:
Looks like it's missing 😄 can you add and test?

Reply:
Sure .